packaging/rhel/syslog-ng.spec

```diff
@@ -153,6 +182,10 @@ Key features:
 Summary: SQL support for %{name}
 Group: Development/Libraries
 Requires: %{name}%{?_isa} = %{version}-%{release}
+Requires: libdbi >= %{libdbi_ver}
+Requires: libdbi-dbd-mysql
```

Comment:
syslog-ng only requires **libdbi**. Drivers are in separate sub-packages for a reason: you do not know what database the user has, and do not want to install hundreds of megabytes of useless dependencies on the user's system.

Reply:
Fixed.